speculos/mcu/seproxyhal.py

```diff
@@ -438,13 +443,26 @@ def handle_finger(self, x: int, y: int, pressed: bool):
         packet += x.to_bytes(2, 'big')
         packet += y.to_bytes(2, 'big')
 
-        self.packet_thread.queue_packet(SephTag.FINGER_EVENT, packet)
+        self.socket_helper.queue_packet(SephTag.FINGER_EVENT, packet)
+
+    def handle_ticker_request(self, action):
```

Comment:
None of the method of this file use this syntax. I rather keep it uniformed. Also because having this kind of information without a `mypy` checker is a great way to have fake information.